src/views/Governance/Governance.tsx

```diff
@@ -53,6 +65,8 @@ const Governance: React.FC = () => {
           <Spacer />
           <Button full text="Off-chain Voting" href="https://snapshot.page/#/yam" variant="tertiary" />
           <Spacer />
+          <RegistrationButton width='125px' />
```

Comment:
thats ok as a first step, but you missed this part for smaller screens such as mobile devices:
> if we hover here, the button changes in size and we dont want that. adding a limited width: 125px; style for the button will fix it, "**keeping in mind smaller screen devices should not have a similar limited width, but should be kept as is currently**"

![small](https://user-images.githubusercontent.com/2432272/107975867-29cc4400-6fd2-11eb-86a7-2b16539a410c.PNG)